scripts/generate_report.sh

```
@@ -0,0 +1,116 @@
+
+set -e
+
+# 環境変数の確認
+if [ -z "$GITHUB_TOKEN" ] || [ -z "$SLACK_WEBHOOK_URL" ] || [ -z "$GITHUB_USERNAME" ]; then
+    echo "Required environment variables are not set"
+    exit 1
+fi
+
+# 過去24時間の期間を設定
+FROM_DATE=$(date -d '24 hours ago' -u +"%Y-%m-%dT%H:%M:%SZ")
+CURRENT_DATE=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
```

Shellcheck (0.10.0)

[warning] CURRENT_DATE appears unused. Verify use (or export if used externally).

(SC2034)